src/python/dxpy/nextflow/nextflow_utils.py

```diff
@@ -7,8 +7,19 @@
 def get_source_file_name():
     return "src/nextflow.sh"
 
-def get_resources_subpath():
-    return "/home/dnanexus/nfp"
+def get_resources_dir_name(resources_dir):
+    """
+    :param resources_dir: Directory with all source files needed to build an applet. Can be an absolute or a relative path.
+    :type resources_dir: str or Path
+    :returns: The name of the folder
+    :rtype: str
+    """
+    if resources_dir == None:
```

Comment:
This is not covered previously. Is there a use case for it?

Reply:
This is for when the resources_dir is no specified (and we want '', for example here: https://github.com/dnanexus/dx-toolkit/pull/824/files#diff-7080b6a8931911206ca88463752bac6d538e2e4b772dbcfaafd1fb2e4ae7e6b6R21

Reply:
Got it. Nice catch!